src/components/AddPeopleAndRoles/OrgPerson.vue

```diff
@@ -0,0 +1,533 @@
+                  />
+                </div>
+                <div v-if="isOrg" class="org-name-container">
+                  <v-text-field
+                    filled
+                    class="item"
+                    label="Full Legal Corporation or Firm Name"
+                    id="firm-name"
+                    v-model="orgPerson.orgName"
+                    :rules="orgNameRules"
+                  />
+                </div>
+                <label class="sub-header">Roles</label>
+                <v-row>
+                  <v-col cols="4" v-if="isPerson">
+                    <v-checkbox v-model="isCompletingParty" label="Completing Party"
+                    :disabled="isRoleLocked(Roles.COMPLETING_PARTY)"
+                    v-bind:class="{'highlightedRole': isRoleLocked(Roles.COMPLETING_PARTY)}"
+                    @change="assignCompletingPartyRole()"/>
+                  </v-col>
+                  <v-col cols="4">
+                    <v-checkbox v-model="isIncorporator"
+                    :label="incorporatorLabel"
+                    :disabled="isRoleLocked(Roles.INCORPORATOR) || orgPerson.type === IncorporatorTypes.CORPORATION"
+                    v-bind:class="{'highlightedRole':
+                    isRoleLocked(Roles.INCORPORATOR) || orgPerson.type === IncorporatorTypes.CORPORATION}"/>
+                  </v-col>
+                  <v-col cols="4" v-if="isPerson">
+                    <v-checkbox v-model="isDirector" label="Director"/>
+                  </v-col>
+                </v-row>
+
+                <label class="sub-header">Mailing Address</label>
+                <div class="address-wrapper">
+                  <base-address
+                    ref="mailingAddressNew"
+                    :editing="true"
+                    :schema="personAddressSchema"
+                    :address="inProgressMailingAddress"
+                    @update:address="updateMailingAddress"
+                    @valid="updateMailingAddressValidity"/>
+                </div>
+
+                <div class="form__row" v-if="isDirector">
+                  <v-checkbox
+                    class="inherit-checkbox"
+                    label="Delivery Address same as Mailing Address"
+                    v-model="inheritMailingAddress"/>
+                  <div v-if="!inheritMailingAddress">
+                    <label class="sub-header">Delivery Address</label>
+                    <div class="address-wrapper">
+                      <base-address
+                        ref="deliveryAddressNew"
+                        :editing="true"
+                        :schema="personAddressSchema"
+                        :address="inProgressDeliveryAddress"
+                        @update:address="updateDeliveryAddress"
+                        @valid="updateDeliveryAddressValidity"/>
+                    </div>
+                  </div>
+                </div>
+
+                <div class="form__row form__btns">
```

Comment:
Check if "large" buttons here match the mockup better. (They seem a little small atm.)